commandline/WSL/install_guide.md

```diff
@@ -1,84 +1,93 @@
 ---
-title: Bash on Ubuntu on Windows - Installation Guide
-description: Installation Guide
-keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu
-author: jackchammons
-ms.date: 9/6/2016
+title: Install Linux Subsystem on Windows 10
+description: Installation instructions for the Linux Subsystem on Windows 10.
+keywords: BashOnWindows, bash, wsl, windows, windows subsystem for linux, windowssubsystem, ubuntu, windows 10
+author: scooley
+ms.date: 7/11/2017
 ms.topic: article
 ms.prod: windows-subsystem-for-linux
 ms.service: windows-subsystem-for-linux
 ms.assetid: 7afaeacf-435a-4e58-bff0-a9f0d75b8a51
 ---
 
-# Installation Guide
+# Windows 10 Installation Guide
 
-> **Important note**  
-	This is the first release of Bash on Windows and it is branded "beta" deliberately - it's not yet complete! You should expect many things to work and for some things to fail! We greatly appreciate you using Bash on Windows and helping us identify the issues we need to fix in order to deliver a great experience.
+## Prerequisites
+Your PC must be running a 64-bit version of Windows 10.
 
-##Prerequisites
-1. Your PC must be running (at a minimum) a 64-bit version of Windows 10 Anniversary Update. **Creators Update is recommended.**
+To find your PC's architecture and Windows build number, open  
+**Settings** > **System** > **About**
 
-> To find your PC's CPU architecture and Windows version/build number, open 
-> **Settings**>**System**>**About**. 
-> Look for the **OS Build** and **System Type** fields.  
+Look for the **OS Build** and **System Type** fields.  
+    ![](media/system.png) 
 
-![](media/system.png) 
+## Install the Windows Subsystem for Linux
+Enable the "Windows Subsystem for Linux" optional feature and reboot.
 
-If your build is below 14393, try checking for updates.
+1. Open PowerShell as Administrator and run:
+    ``` PowerShell
+    Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+    ```

```

Comment:
Alternatively, they can go to "Turn Windows feature on or off" and enable the "Windows Subsytem for Linux" feature.